src/JsonHandler.php

```diff
@@ -11,11 +11,11 @@ trait JsonHandler
      * Handle the json response. Check if exception is treated. If true call
      * the specific handler. If false set the default response to be returned.
      *
-     * @param  \Exception $exception
+     * @param  \Throwable  $exception
      *
      * @return \Illuminate\Http\JsonResponse
      */
-    public function jsonResponse(Exception $exception)
+    public function jsonResponse(Throwable $exception)
```

Comment:
@designvoid
wouldn't it be necessary to import the Throwable class?

Reply:
Please, can you remove the unused Exception import?

Reply:
Sure thing - that should be fixed now - literally no idea how I missed that